service/history/task/transfer_task_executor_base.go

```diff
@@ -99,6 +103,9 @@ func (t *transferTaskExecutorBase) pushActivity(
 		t.logger.Fatal("Cannot process non activity task", tag.TaskType(task.GetTaskType()))
 	}
 
+	// Ratelimiting is not done. This is only to count the number of requests via metrics
+	t.wfIDCache.AllowInternal(task.DomainID, task.WorkflowID)
```

Comment:
This function is called by transfer active task executor and transfer standby task executor. Are we going to count those together in same rate limit quota?

Reply:
Thats a good catch. I have considered to address this in a follow up PR